api/http/handler/handler.go

```diff
@@ -243,6 +245,10 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		http.StripPrefix("/api", h.WebhookHandler).ServeHTTP(w, r)
 	case strings.HasPrefix(r.URL.Path, "/storybook"):
 		http.StripPrefix("/storybook", h.StorybookHandler).ServeHTTP(w, r)
+	case strings.HasPrefix(r.URL.Path, "/metrics"):
+		if h.MetricsHandler != nil {
+			h.MetricsHandler.ServeHTTP(w, r)
+		}
```

Comment:
Not sure what would be the behaviour, when the the feature flag is off. Will it try to return a file??
Maybe instead we could move the feature flag check to the metrics handler so that it'd return 404 unless the flag is on? That would also allow let enabling/disabling metrics without restarting Portainer, if we'd have a way to set feature flags in runtime.